browser/youtube_script_injector/android_youtube_script_injector_browsertest.cc

```diff
@@ -0,0 +1,98 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include <string>
+
+#include "base/path_service.h"
+#include "brave/components/constants/brave_paths.h"
+#include "chrome/test/base/android/android_browser_test.h"
+#include "chrome/test/base/chrome_test_utils.h"
+#include "chrome/test/base/testing_browser_process.h"
+#include "content/public/common/content_client.h"
+#include "content/public/test/browser_test.h"
+#include "content/public/test/browser_test_utils.h"
+#include "content/public/test/content_mock_cert_verifier.h"
+#include "net/dns/mock_host_resolver.h"
+#include "net/test/embedded_test_server/embedded_test_server.h"
+
+namespace {
+const char* command = "window._pipScriptInjected === true";
```

Comment:
`constexpr` and should be kCheckScriptInjected or something https://google.github.io/styleguide/cppguide.html#Constant_Names